### ISSUE TITLE
Column 'resourcecontainer' exceeds MaxLength limit

### ISSUE THREAD

**Issue by sybaris** [MarkMpn/Sql4Cds#302]
Hi,

With this query :

```
select * 
from workflow 
where workflowidunique='5417bfbf-b674-4277-8eaa-1a3a82e66439'

```
I have the following error : Column 'resourcecontainer' exceeds MaxLength limit

![image](https://github.com/MarkMpn/Sql4Cds/assets/11009997/8e54a56f-d216-4c22-b786-a084baba3472)

Regards
Sybaris

**Comment by MarkMpn** [comment]
I can't reproduce this one because none of the records in my test system have a value in this column. I think it will be fixed in the next update though.

**Comment by sybaris** [comment]
Hi. I was in 7.1.0 It seems to be solved in 7.2.2
Thanks for your answer...
But I have another issue : 
![image](https://github.com/MarkMpn/Sql4Cds/assets/11009997/825df53c-c2ee-45be-ab8a-264459da87ad)
